### DISCUSSION TITLE
Make Stream - Public

### DISCUSSION THREAD

**Discussion by FirstUpRadio** [AzuraCast/AzuraCast#6391] [Q&A]
I Can't Figure Out How To Make My Stream Public

![Screenshot 2023-07-03 235301](https://github.com/AzuraCast/AzuraCast/assets/138549163/38ad9950-e98e-48d7-a35b-131afcae97d3)

**Comment by BusterNeece** [comment]
@FirstUpRadio Public in what sense? Public as in, advertised in Yellow Pages directories? You can do that via the `Broadcasting` menu, then the `Mount Points` sub-item.

The reason you're not able to change that is because the reseller you're using didn't give you permission to see that item. You should follow up with them about that.

**Reply by FirstUpRadio** [reply]
Thank You!
